src/clients/apiClient.ts

```diff
@@ -145,6 +145,17 @@ export async function fetchLogin(encodedUsernamePassword: string): Promise<void>
 	}
 }
 
+export async function fetchIsAdmin(encodedUsernamePassword: string): Promise<void> {
```

Comment:
Can we return a boolean here? Then we can set the `isAdmin` variable to the response of this method
```suggestion
export async function fetchIsAdmin(encodedUsernamePassword: string): Promise<boolean> {
```